projects/demo/src/pages/kit/number/examples/maskito-parse-number-demo.md

```diff
@@ -0,0 +1,8 @@
+```ts
+import {maskitoParseNumber} from '@maskito/kit';
+
+maskitoParseNumber(
+  '10 000,42', // document.querySelector('input').value
```

Comment:
what do you think about is it can be dependent on locale?

Reply:
Reg Exp inside this utility parses any custom `thousandSeparator`.

This utility requires only the second arg – `decimalSeparator`.